# name of the room
string room_name

# ID of the room
int8 room_ID

# level of charge of the robot battery
int8 charge_level